### ISSUE TITLE
Please add a config to weight eggs in chests

### ISSUE THREAD

**Issue by CurePindal** [ata4/dragon-mounts#9]
Hello Barracuda!

Can you please add a config to weight the eggs in the chests? 
When we use other plugins that modify chests and even spawn structures that contain generic chests the eggs get totally out of hand. Like 10+ eggs per hour while exploring!

An option to adjust this per config would be very welcome! =)

**Comment by SupernovaCo** [comment]
Yeah. This would be extremely handy. Or maybe add some sort of generated structure "dragon nest" with one or two adult wild dragons and some dragon eggs.
